src/js/base/module/Fullscreen.js

```diff
@@ -32,23 +33,27 @@ export default class Fullscreen {
    */
   toggle() {
     this.$editor.toggleClass('fullscreen');
-    if (this.isFullscreen()) {
+    const isFullscreen = this.isFullscreen();
+    this.$scrollbar.toggleClass(this.scrollbarClassName, isFullscreen);
```

Comment:
What is the `isFullscreen` doing here, when I remove it, and rebuild, then the classes are applied to the html and body as intended?

Reply:
https://api.jquery.com/toggleclass/#toggleClass-className-state

Reply:
Ah ok, I should have realised, I was thinking it should be adding the class regardless.